lib/fixture/fixture.ex

```diff
@@ -18,9 +18,9 @@ defmodule ElixirTools.Fixture do
   @doc """
   Looks for a fixture without decoding
   """
-  @spec load!(String.t(), String.t()) :: String.t()
-  def load!(fixture, location \\ @fixture_location) do
-    file = location <> fixture <> ".json"
+  @spec load!(String.t(), String.t(), String.t()) :: String.t()
+  def load!(fixture, location \\ @fixture_location, file_extension \\ ".json") do
+    file = location <> fixture <> file_extension
```

Comment:
since load has nothing to do with json, I would prefer moving this to load_json! https://github.com/digitalorigin/pg-elixir-tools/pull/56/files#diff-0dbd60fba723e3e21c6dff1ddf34eeb4R11

Reply:
Maybe for another PR @hylkealons ?

Reply:
since it's directly related, I'd do it here: but up to @fartek

Reply:
The reason why I **would leave the default .json here** is because maybe some modules in some projects are depending on this default functionality. This is a suggestion to break the current API, which I don't wanna do (in this PR).